### PR TITLE
Adds more to the Create Dataset flow

### LINKED ISSUE #6: Create a dataset flow
This should be the same as the dguproto1 flow, but include the extra metadata we want to collect.

### PR DESCRIPTION
**Pull request by rossjones**
Still some questions around:
1. How do we handle multiple file upload
2. Do we allow skipping irrelevant steps?

Fixes #6

### CODE REVIEW

File: app/views/manage_data/upload_new_dataset/file_upload.html [hunk 1]
```diff
@@ -20,57 +20,53 @@
...
-
-      <form method="get" action="/create_account/email_sent" class="form">
-
-      {{formData | safe}}
+      <div class="column-two-thirds">
 
         <h1 class="heading-large">
-          Upload a new dataset
+        Upload your file
         </h1>
```
Comment (maxf) [inline app/views/manage_data/upload_new_dataset/file_upload.html:29]: Should span the whole width, ie be moved outside of the grid-row

File: app/views/manage_data/upload_new_dataset/geo.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    <div class="column-two-thirds">
+
+      <form method="get" action="file_upload" class="form">
+
+      {{formData | safe}}
+
+        <h1 class="heading-large">
+          Choose a country
+        </h1>
```
Comment (maxf) [inline app/views/manage_data/upload_new_dataset/geo.html:34]: move at the top

File: app/views/manage_data/upload_new_dataset/licence.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+    <div class="column-two-thirds">
+
+      <form method="get" action="themes.html" class="form">
+
+      {{formData | safe}}
+
+        <h1 class="heading-large">
+          Choose a licence
+        </h1>
```
Comment (maxf) [inline app/views/manage_data/upload_new_dataset/licence.html:34]: as above

File: app/views/manage_data/upload_new_dataset/period.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+      {{formData | safe}}
+
+        <h1 class="heading-large">
+          Date
+        </h1>
+
+        <p>What time period does this dataset cover?</p>
+
+        <br>
```
Comment (maxf) [inline app/views/manage_data/upload_new_dataset/period.html:37]: <br/>

**Review by maxf — CHANGES_REQUESTED** [review]

File: app/views/manage_data/upload_new_dataset/file_upload.html (hunk 1 above)
Comment (rossjones) [reply app/views/manage_data/upload_new_dataset/file_upload.html:29]: Should do this as part of cleanup

**Review by maxf — APPROVED** [review]

**Pull request merged**